### PR TITLE
Release 11.1.1

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Is this the right process? The docs in (https://github.com/theforeman/foreman-installer-modulesync/blob/master/README.md) weren't clear to me how to handle a branched release. I also have trouble getting the scrpits to work on my Fedora with Ruby 2.7.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [11.1.1](https://github.com/theforeman/puppet-foreman_proxy_content/tree/11.1.1) (2020-08-27)
```
Comment (ekohl) [inline CHANGELOG.md:3]: Can you include a `Full Changelog` link like done for the others? Once the tag shows up, it should work.
```suggestion
## [11.1.1](https://github.com/theforeman/puppet-foreman_proxy_content/tree/11.1.1) (2020-08-27)

[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy_content/compare/11.1.0...11.1.1)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+
+**Fixed bugs:**
+
+- Fixes [\#30716](https://projects.theforeman.org/issues/30716) - Ensure /pub on foreman proxy can be browsed by default [\#278](https://github.com/theforeman/puppet-foreman_proxy_content/pull/278) ([ehelms](https://github.com/ehelms))
+
 ## [11.1.0](https://github.com/theforeman/puppet-foreman_proxy_content/tree/11.1.0) (2020-06-30)
 
 [Full Changelog](https://github.com/theforeman/puppet-foreman_proxy_content/compare/10.0.0...11.1.0)
```
Comment (ekohl) [inline CHANGELOG.md:11]: I just noticed this is wrong. Mind changing it while you're at it?
```suggestion
[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy_content/compare/11.0.0...11.1.0)
```

**Review by ekohl — CHANGES_REQUESTED** [review]
Pretty much, but would you mind squashing into a single commit? The docs don't mention the arguments that you need to pass in. Since the scripts don't implement a `--help`, this is hard to detect but they're all short enough that reading the source is reasonable. https://github.com/theforeman/foreman-installer-modulesync/pull/127 does give some examples.

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Test failures can be ignored - for the stable branch we should be pinning fixtures to non-master versions but I believe the current code will work.

**Pull request merged**